tests/perf/run_perf_tests.py

```diff
@@ -7,69 +7,80 @@
 import os
 import csv
 import sys
+import getopt
 from datetime import datetime
 
 DEFAULT_TEST_SEQUENCE = "50:30,200:30,200:60,400:30"
 DEFAULT_REPETITIONS = 10
-DEFAULT_VEGETA_PATTERN_TAR_FILE = ""
-DEFAULT_DAEMON_ON_CORE = "-"
-DEFAULT_GETH_HOME_DIR = "../../../turbo-geth/"
+DEFAULT_VEGETA_PATTERN_TAR_FILE = "turbo_geth_stress_test_001.tar"
+DEFAULT_DAEMON_VEGETA_ON_CORE = "-:-"
 DEFAULT_TG_ADDRESS = "localhost:9090"
+DEFAULT_GETH_BUILD_DIR = "../../../turbo-geth/build/"
+DEFAULT_SILKRPC_BUILD_DIR = "../../build_gcc_release/"
 
+VEGETA_REPORT = "vegeta_report.hrd"
+VEGETA_PATTERN_SILKRPC = "/tmp/turbo_geth_stress_test/vegeta_geth_eth_getLogs.txt"
+VEGETA_PATTERN_RPCDAEMON = "/tmp/turbo_geth_stress_test/vegeta_turbo_geth_eth_getLogs.txt"
+
+def usage(argv):
+    """ Print script usage
+    """
+    print("Usage: " + argv[0] + " -h -p vegetaPatternTarFile -c daemonOnCore  -t turboGethAddress -g turboGethHomeDir -s silkrpcBuildDir -r testRepetitions - t testSequence")
+    print("")
+    print("Launch an automated performance test sequence on Silkrpc and RPCDaemon using Vegeta")
+    print("")
+    print("-h                      print this help")
+    print("-p vegetaPatternTarFile path to the request file for Vegeta attack")
+    print("-c daemonVegetaOnCore   cpu list in taskset format for daemon & vegeta (e.g. 0-1:2-3 or 0-2:3-4 or 0,2:3,4...) [default: " + DEFAULT_DAEMON_VEGETA_ON_CORE +"]")
+    print("-t turboGethAddress     address of TG Core component as <address>:<port> (e.g. localhost:9090)           [default: " + DEFAULT_TG_ADDRESS + "]")
+    print("-g turboGethHomeDir     path to TG home folder (e.g. ../../../turbo-geth/)                               [default: " + DEFAULT_GETH_BUILD_DIR + "]")
+    print("-s silkrpcBuildDir      path to build home folder (e.g. ../../build_gcc_release/)                        [default: " + DEFAULT_SILKRPC_BUILD_DIR + "]")
```

Comment:
`path to build home folder` -> `path to Silkrpc build folder`

Reply:
ok